.github/workflows/expectfail.sh

```diff
@@ -1,8 +1,7 @@
 #!/bin/bash
 
 if $1; then
-    echo "FALSE POSITIVE! Expected failure but succeeded"
-    exit 1
+    echo "Succeeded"
```

Comment:
You could just remove the `bash .github/workflows/expectfail.sh` from the yml (and delete this file), instead of removing the `exit 1` here.